addons/mrp_subcontracting/__manifest__.py

```diff
@@ -93,7 +93,8 @@
             'web/static/lib/bootstrap/js/dist/scrollspy.js',
             'web/static/lib/bootstrap/js/dist/tab.js',
             'web/static/lib/bootstrap/js/dist/toast.js',
-            'web/static/src/libs/bootstrap.js',
+            'web/static/src/libs/bootstrap/bootstrap.js',
+            'web/static/src/libs/bootstrap/bootstrap.scss',            
```

Comment:
I don't know what this change is about but surely it either deserves its own commit and comments or it's just wrong. There is a call to `_assets_bootstrap` a few lines below, surely this is already including bootstrap css?

Same for everywhere else you added this